components/empty-state/empty-state-styles.js

```diff
@@ -0,0 +1,61 @@
+import '../colors/colors.js';
+import { css } from 'lit';
+
+export const emptyStateStyles = css`
+
+	:host {
+		border: 1px solid var(--d2l-color-mica);
+		border-radius: 6px;
```

Comment:
If the design intent here was to match the radius of our buttons, we actually use `0.3rem` -- which is equivalent to `6px` except that if the user is using a larger font size, it'll be a larger radius.